core/templates/dev/head/pages/preferences/preferences.html

```diff
@@ -231,25 +231,25 @@
             <div class="col-lg-10 col-md-10 col-sm-10">
               <div class="checkbox">
                 <label>
-                  <input type="checkbox" ng-model="canReceiveEmailUpdates" ng-change="saveEmailPreferences(canReceiveEmailUpdates, canReceiveEditorRoleEmail, canReceiveFeedbackMessageEmail, canReceiveSubscriptionEmail)">
+                  <input type="checkbox" class="protractor-test-canReceiveEmailUpdates" ng-model="canReceiveEmailUpdates" ng-change="saveEmailPreferences(canReceiveEmailUpdates, canReceiveEditorRoleEmail, canReceiveFeedbackMessageEmail, canReceiveSubscriptionEmail)">
```

Comment:
Formatting for CSS class names is always hyphenated, in this codebase, and it's also important for the classes to be descriptive of their elements. So, here, you could do protractor-test-email-updates-checkbox or similar. The "checkbox" bit is important.

Ditto below.